.github/workflows/e2e_test_ci.yml

```diff
@@ -621,12 +622,14 @@ jobs:
         if: always()
         working-directory: duckdb-app
         run: |
-          killall spice
+          killall spice || true
           cat spice.log
 
   test_local_acceleration:
     name: 'Test acceleration on ${{ matrix.target.name }} using ${{ matrix.acceleration.engine }}'
     runs-on: ${{ matrix.target.runner }}
+    env:
+      HOME: ${{ github.workspace }}
```

Comment:
Why is the `env: HOME: ${{ github.workspace }}` is used instead of `HOME=$GITHUB_WORKSPACE` here

Reply:
Moved on job level so Spice runtime (spiced) is installed/placed to the right folder. GITHUB_WORKSPACE was added due to write permission issue. We will be able to remove this as we've switched where spice data folder is located (not at HOME anymore)